src/Popover/PopoverContent.js

```diff
@@ -1,13 +1,10 @@
 import styled from '@emotion/styled';
-import { transparentize } from 'polished';
 
 import Card from '../Card';
 
 const PopoverContent = styled(Card)`
   background: ${props => props.theme.colors.white};
-  box-shadow: 0 4px 8px
-      ${props => transparentize(0.9, props.theme.colors.black)},
-    0 0 1px ${props => props.theme.colors.grey20};
+  box-shadow: ${props => props.theme.boxShadows.elevation2};
```

Comment:
If it works, it might be better to set `boxShadow` defaultProp on `PopoverContent` so that we can override it if ever wanted to.

Reply:
no luck :/

Reply:
not sure how to set a default prop from the theme when the theme itself is another prop

Reply:
I was thinking something like:
```
PopoverContent.defaultProps = {
  ...Card.defaultProps,
  boxShadow: 'elevation2',
};
```

Since this is just internal though, I'm not sure if it matters much.